packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -1632,3 +2130,335 @@ Specific property types support a subset of [JSON Schema validations](https://to
+    "base": {
+      "id": "#base",
+      "properties": {
+        "description": {
+          "description": "Description",
+          "master": {
+            "type": "PROFILE_MASTER"
+          },
+          "maxLength": 1024,
+          "mutability": "READ_WRITE",
+          "permissions": [
+            {
+              "action": "READ_WRITE",
+              "principal": "SELF"
+            }
+          ],
+          "scope": "NONE",
+          "title": "Description",
+          "type": "string"
+        },
+        "name": {
+          "description": "Name",
+          "master": {
+            "type": "PROFILE_MASTER"
+          },
+          "maxLength": 255,
+          "mutability": "READ_WRITE",
+          "permissions": [
+            {
+              "action": "READ_WRITE",
+              "principal": "SELF"
+            }
+          ],
+          "required": true,
+          "scope": "NONE",
+          "title": "Name",
+          "type": "string"
+        }
+      },
+      "required": [
+        "name"
+      ],
+      "type": "object"
+    },
+    "custom": {
+      "id": "#custom",
+      "properties": {
+        "exampleCustomProperty": {
+          "description": "exampleCustomProperty",
+          "master": {
+            "type": "PROFILE_MASTER"
+          },
+          "maxLength": 20,
+          "minLength": 1,
+          "mutability": "READ_WRITE",
+          "permissions": [
+            {
+              "action": "READ_WRITE",
+              "principal": "SELF"
+            }
+          ],
+          "required": true,
+          "scope": "NONE",
+          "title": "exampleCustomProperty",
+          "type": "string",
+          "unique": "UNIQUE_VALIDATED"
+        }
+      },
+      "required": [
+        "exampleCustomProperty"
+      ],
+      "type": "object"
+    }
+  }
+}
+```
+
+#### Group Profile base subschema
+
+All Okta-defined Profile properties are defined in a Profile subschema with the resolution scope `#base`. These properties can't be removed or edited.  Attempting to do so anyway will have no effect.
+
+The base Group Profile properties are as follows:
+
+| Property          | Description                                                                                                                  | DataType | Nullable | Unique | Readonly | MinLength | MaxLength |
+| ----------------- | ---------------------------------------------------------------------------------------------------------------------------- | -------- | -------- | ------ | -------- | --------- | --------- |
+| name              | unique identifier for the Group                                                                                              | String   | FALSE    | TRUE   | FALSE    |           | 1024      |
+| description       | human readable description of the Group                                                                                      | String   | TRUE     | FALSE  | FALSE    |           | 1024      |
```

Comment:
description > Description
human... > Human...

Reply:
updated to "Human"; `description` is a property